documentation/general/program-scale-testing-requirements/hero-implementation-details.md

```diff
@@ -0,0 +1,261 @@
+# Hero test from theory to practice
+
+## Introduction
+
+Hero test is a synthetic test designed to stress the device under the worst-case scenario. Reality is that worst case scenario for device A need not always equal the worst case for device B. Moreover, vendors aim to present both the best and worst cases, not only the worst ones. Since the predicted performance in production will likely fall somewhere between the best- and worst-case scenarios, knowing the lower and upper bounds will give a solid idea of how well the device performs. This makes knowing both the best- and worst-case scenarios valuable.
+
+For this reason, the Hero Test implementation is actually a collection of tests that gather multiple datapoints.
+
+As philosophy goes for hero test suite of tests i believe it is important not to provide a PASS/FAIL but a characterization of the hardware with multiple data points and let each individual decide based on data if the hardware is suitable for their deployment scenario or not. (functionality is assumed to be working,  and purpose here is to asses performance)
```

Comment:
>(functionality is assumed to be working, and purpose here is to asses performance)

Better:
(Functionality is assumed to be working, and the purpose here is to assess performance.)